<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Towel 42 Development, LLC - CMathJaxQt6 GUI Example</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="5" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>MathJaxQt6 Widget Supports Wheel Zoom</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDoubleSpinBox" name="minScale">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="syncRender">
      <property name="text">
       <string>Synchronous Render</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Current Scale:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QCheckBox" name="hideEmptyOrInvalid">
      <property name="text">
       <string>Hide when Empty or Invalid?</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QDoubleSpinBox" name="scale">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QCheckBox" name="autoUpdateMinimumParentHeight">
      <property name="text">
       <string>Auto update Minimum Parent Height?</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>Enter TeX Here</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QDoubleSpinBox" name="maxScale">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Minimum Scale:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QPushButton" name="asyncRender">
      <property name="text">
       <string>Render</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="8">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>MathJaxQt6 Widget:</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="NTowel42::CMathJaxQt6Widget" name="mathJaxWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>SVG Code:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="svgCode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>&quot;Hidden&quot; QWebEngineView owned by the Rendering Engine:</string>
       </property>
       <layout class="QVBoxLayout" name="webEngineViewLayout"/>
      </widget>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Maximum Scale:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Pixels Per Formula:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="pixelsPerFormula">
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="3">
     <widget class="QCheckBox" name="autoSizeToParentWidth">
      <property name="text">
       <string>Auto Size to Parent Width?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NTowel42::CMathJaxQt6Widget</class>
   <extends>QWidget</extends>
   <header>T42-MathJaxQt6/include/MathJaxQt6Widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>asyncRender</tabstop>
  <tabstop>syncRender</tabstop>
  <tabstop>autoUpdateMinimumParentHeight</tabstop>
  <tabstop>hideEmptyOrInvalid</tabstop>
  <tabstop>minScale</tabstop>
  <tabstop>maxScale</tabstop>
  <tabstop>scale</tabstop>
  <tabstop>svgCode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
